src/txmempool.cpp

```diff
@@ -197,13 +197,15 @@ util::Result<CTxMemPool::setEntries> CTxMemPool::CalculateAncestorsAndCheckLimit
 }
 
 bool CTxMemPool::CheckPackageLimits(const Package& package,
+                                    const std::vector<int64_t> &all_tx_vsizes,
```

Comment:
in fef601e2a4d790781fee316c1a55ea48a134a0f1
Given that this function is only interested in the total size, why not just pass in a `int64_t total_vsize`?

Reply:
> why not just pass in a int64_t total_vsize?

Indeed, then you don't have to gather all the vsizes twice as well.

Reply:
Done